content/200-concepts/100-components/01-prisma-schema/03-generators.mdx

```diff
@@ -53,6 +55,3 @@ generator client {
 
 In that case, Prisma detects your operating system and finds the right binary file for it based on the [list of supported operating systems](../../../reference/api-reference/prisma-schema-reference#binarytargets-options) <span class="api"></span>. As you're running Mac OS, the binary file that was compiled for `darwin` will be selected.
 
-### Custom binary targets
```

Comment:
@janpio Was this wrong, or just not the right place?

Reply:
That was "wrong". It is closely related to this page, but generally describes something different. This linked to a description of how you can bring your own binaries - then "binary targets" are not relevant any more at all - Prisma just uses _that_ file. The sentence describes that correctly (but it is not related to this page then), the headlined did not.

There actually is an env var for the binary targets as well, but that has the same effect as setting `binaryTargets` inside `generator` as describe further up the page. There it could theoretically also explain and link to that env var.